fvm/evm/types/events.go

```diff
@@ -143,7 +143,7 @@ func (p *TransactionExecutedPayload) CadenceEvent() (cadence.Event, error) {
 			cadence.String(p.Result.VMErrorString()),
 			cadence.NewUInt8(p.Result.TxType),
 			cadence.NewUInt64(p.Result.GasConsumed),
-			cadence.String(hex.EncodeToString(p.Result.DeployedContractAddress.Bytes())),
```

Comment:
is this for 0x prefixing?

Reply:
Not only for that. The format changes:
from: 99466ed2e37b892a2ee3e9cd55a98b68f5735db2
to: 0x99466ED2E37B892A2Ee3E9CD55a98b68f5735db2
which is the preferred check-summed variant.